colossus-metrics/src/main/scala/colossus/metrics/Collection.scala

```diff
@@ -197,11 +192,16 @@ object LocalCollection {
  * events just as function calls), this should not be used for very-high
  * frequency events.
  */
-class SharedCollection(val local: LocalCollection) extends Collection[SharedLocality] {
+private[colossus] class SharedCollection(val local: LocalCollection, val collector: ActorRef) extends Collection[SharedLocality] {
```

Comment:
Is this private modifier here just to prevent external application code from invoking this constructor?